### DISCUSSION TITLE
unknown option: --marker=*

### DISCUSSION THREAD

**Discussion by derVedro** [PatrickF1/fzf.fish#149] [Q&A] [answered]
Hey, I wanted to try that all, but if search something with ctrl + f I get a `unknown option: --marker=*` in my shell. If I remove the _marker_ option from here https://github.com/PatrickF1/fzf.fish/blob/20ec3e4eea8e5c9e8c9d156bd9d4270c3b61ba46/conf.d/fzf.fish#L36
it works. May be some ubuntu 20.04 stuff is too outdated?

I use the last fish 3.2.1 from a PPA, but fzf, fd and bat are from vanilla repos:
fd 7.4.0
fzf 0.20.0
bat 0.12.1

**Comment by PatrickF1** [comment, marked as answer]
Hi @derVedro! Great question! Your fzf is from before the --marker flag was introduced. I'd upgrade to at least 0.21 (https://github.com/junegunn/fzf/blob/master/CHANGELOG.md#0210). I should add the fzf version as a system requirement in the installatino.

**Reply by chaitan94** [reply]
For Ubuntu 20.04 users like the OP who have 0.20 since they installed fzf from apt, here are the quick steps to upgrade:

```sh
# Install from apt
sudo apt remove fzf

# Install via git clone
git clone --depth 1 https://github.com/junegunn/fzf.git ~/.fzf
~/.fzf/install
```

**Reply by rorynolan** [reply]
This installs `fzf` to `~/fzf/bin`. I had to copy the executable to a location on my PATH `(/usr/local/bin`) for it to work with fzf.fish.

**Reply by PatrickF1** [reply]
apt should be way past 0.20 by now so you can just `apt install fzf`

**Reply by rorynolan** [reply]
That's true. I'm on an older Ubuntu though so I had to go the github route.
